src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_x509chain.cpp

```diff
@@ -186,7 +186,8 @@ static void MergeStatusCodes(CFTypeRef key, CFTypeRef value, void* context)
         // It doesn't represent a new error code, and we're still getting the old ones, so
         // just ignore it for now.
     }
-    else if (CFEqual(keyString, CFSTR("NonEmptySubject")) || CFEqual(keyString, CFSTR("GrayListedKey")))
+    else if (CFEqual(keyString, CFSTR("NonEmptySubject")) || CFEqual(keyString, CFSTR("GrayListedKey")) ||
+             CFEqual(keyString, CFSTR("CTRequired")))
```

Comment:
Might be worth a comment indicating what `CTRequired` actually means, now you've satisfied yourself that it's not relevant to us.